force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -62,14 +66,28 @@ public with sharing class ClassesObjectsMethods {
      */
     public static List<Boolean> checkStringContent(String input) {
         // Initialize a list to store the results
+        List<Boolean> results = new List<Boolean>();
         
         // Check if the string starts with 'Sales'
+        if(input.substring(0,5) == 'Sales') {
+            results.add(true);
+        } else {
+            results.add(false);
+        }
         
         // Check if the string ends with 'Force'
-        
+        if (input.substring(input.length()-5,input.length()) == 'Force') {
+            results.add(true);
+        } else {
+            results.add(false);
+        }
         // Check if the string contains 'Awesome'
-        
-        return null; // Replace null with the variable you used to store the result
+        if(input.contains('Awesome')) {
+            results.add(true);
+        } else {
+            results.add(false);
+        }
+        return results; 
```

Comment:
I encourage you to simplify this. Expressions using operators like = or contains return Booleans.  You can solve this without any if / else logic.